tests/integration/tier0/inhibit-if-kmods-is-not-supported/test_kmods_not_supported.py

```diff
@@ -61,16 +75,43 @@ def test_do_not_inhibit_if_module_is_not_loaded(shell, convert2rhel):
             c2r.sendline("y")
             prompt_amount -= 1
 
-        assert c2r.expect("Kernel modules are compatible.") == 0
+        # Stop conversion before the point of no return as we do not need to run the full conversion
+        # with the check for kompatible kernel modules being right before the PONR.
+        assert c2r.expect("All loaded kernel modules are available in RHEL.") == 0
         c2r.expect("Continue with the system conversion?")
         c2r.sendline("n")
         assert c2r.exitstatus != 0
 
 
-def test_tainted_kernel_inhibitor(shell, convert2rhel):
-    # This test marks the kernel as tainted which is not supported by convert2rhel.
+def test_inhibit_if_module_is_force_loaded(shell, convert2rhel):
+    """
+    Test force loads kmod and checks for Convert2RHEL not being inhibited.
```

Comment:
I'm confused. The conversion should be inhibited no?

Reply:
A force loaded kernel module is considered tainted as well and the conversion is supposed to be inhibited as well.

Reply:
Oh yeah, I have changed the function name and did not apply the changes to the docstring